build-tools/packages/build-cli/src/commands/generate/buildVersion.ts

```diff
@@ -102,13 +109,27 @@ export default class GenerateBuildVersionCommand extends BaseCommand<
 		}
 
 		// Generate and print the version to console
-		const simpleVersion = getSimpleVersion(
+		let simpleVersion = getSimpleVersion(
 			fileVersion,
 			flags.build,
 			isRelease,
 			useSimplePatchVersion,
 		);
-		const version = useTestVersion ? `0.0.0-${flags.build}-test` : simpleVersion;
+
+		if (isRelease && isAlphaOrBetaTypes) {
+			this.errorLog(`ERROR: alpha/beta prereleases are allowed`);
```

Comment:
Is this error message inverted? Should it say that releases can't be alpha/beta?